EventFilter/L1TRawToDigi/python/caloStage1Digis_cfi.py

```diff
@@ -3,15 +3,8 @@
 caloStage1Digis = cms.EDProducer(
     "L1TRawToDigi",
     Setup = cms.string("stage1::CaloSetup"),
-    #InputLabel = cms.InputTag("l1tDigiToRaw"),
-    InputLabel = cms.InputTag("rawDataCollector"),
```

Comment:
Why do you remove the InputTag denoting the  input product - recall the PR where I had to put it back in on some other module. Each and every parameter must be in fillDescriptions, and with a value!